tests/util.go

```diff
@@ -118,6 +130,10 @@ func rekorServer() string {
 	return "http://localhost:3000"
 }
 
+func coverageFlag() string {
+	return "-test.coverprofile=/tmp/rekor-cli."+randomSuffix(8)+".cov"
```

Comment:
is there a way to ignore `pkg/generated`?

Reply:
I think I can strip the generated files from the coverage file before uploading to codecov